packages/logger/src/LoggerFactory.ts

```diff
@@ -0,0 +1,31 @@
+import { GCPStructuredLog } from "./structuredLog/GCPStructuredLog";
+import { LoggerOptions } from "./interfaces/LoggerOptions";
+import { createConsolaStructuredLogger } from "./ConsolaStructuredLogger";
+
+export enum LoggerType {
+  ConsolaGcp = "consola-gcp",
+}
+
+/**
+ * Creates a logger based on the type
+ * Available types:
+ * - consola-gcp (Consola logger with GCP structured log)
+ *
+ * @param type The type of logger to create
+ */
+export class LoggerFactory {
+  static create(
+    type: LoggerType,
+    options: LoggerOptions = {
```

Comment:
Maybe we should merge default options with those that are passed through params?

Reply:
ok, fixed